helm/charts/kratos/templates/job-migration.yaml

```diff
@@ -44,6 +44,10 @@ spec:
 {{ toYaml .Values.deployment.extraVolumeMounts | indent 10 }}
          {{- end }}
       restartPolicy: Never
+      {{- if .Values.deployment.extraInitContainers }}
+      initContainers:
+{{ tpl .Values.deployment.extraInitContainers . | indent 8 }}
```

Comment:
I think the tpl here is used in cases you need to pass data into the init container. This would allow you to specify for example `{{ .Release.Namespace }}` as a value for some env variable in the container. But i agree, in some cases this could lead to invalid parsing. Maybe it would be best to encapsulate it like `toYaml ( tpl .Values.deployment.extraInitContainers . ) | indent 8` ?

Reply:
Great tip!